.github/workflows/pr.yaml

```diff
@@ -26,14 +26,17 @@ jobs:
           # Version Spec of the version to use.  Examples: 12.x, 10.15.1, >=10.15.0
           node-version: ^12.4.0
 
-      - name: Install Deps Ubuntu
-        run: sudo apt-get -y install libkrb5-dev libsecret-1-dev net-tools libstdc++6 gnome-keyring
+      # - name: Install Deps Ubuntu
+      #   run: sudo apt-get -y install libkrb5-dev libsecret-1-dev net-tools libstdc++6 gnome-keyring
 
-      - name: Install Dependencies
-        run: npm ci
+      - name: Not Implemented
+        run: echo "GitHub Actions workflows are disabled for now. To be fixed in https://jira.mongodb.org/browse/COMPASS-4788"
```

Comment:
See also https://github.com/mongodb-js/compass/pull/2174#issuecomment-829264752. Opened a ticket for that https://jira.mongodb.org/browse/COMPASS-4788